src/sage/numerical/optimize.py

```diff
@@ -532,7 +532,7 @@ def minimize_constrained(func, cons, x0, gradient=None, algorithm='default', **a
         sage: f(y,x) = x - y
         sage: c1(y,x) = x
         sage: c2(y,x) = 1-y
-        sage: minimize_constrained(f, [c1, c2], (0,0))
+        sage: minimize_constrained(f, [c1, c2], (0,0)) # abs tol 1e-04
```

Comment:
So this one is not random? But lack precision instead (mind you I think more precision testing are too high as a physicist).

Reply:
That one does indeed have a minimum at (1,0), but the second coordinate is given as 5e-05 with the new implementation

Reply:
Cool with me.